conda/post-link.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+echo "Initializing for Zowe install" > $PREFIX/.messages.txt
+
+# Before attempting install of plugin into Zowe, must first gather relevant environment variables
+# This is not guaranteed, and will fail if the user has not previously specified the location of Zowe
+
+if [ -n "$ZOWE_ROOT_DIR" ]; then
+  if [ -d "${ZOWE_ROOT_DIR}/components/app-server" ]; then
+      ZLUX_ROOT=$ZOWE_ROOT_DIR/components/app-server/share
+  elif [ -d "${ZOWE_ROOT_DIR}/zlux-app-server" ]; then
+      ZLUX_ROOT=$ZOWE_ROOT_DIR
+  fi
+elif [ -n "$ROOT_DIR" ]; then
+  if [ -d "${ROOT_DIR}/components/app-server" ]; then
+    ZLUX_ROOT=$ROOT_DIR/components/app-server/share
+  elif [ -d "${ROOT_DIR}/zlux-app-server" ]; then
+    ZLUX_ROOT=$ROOT_DIR  
+  fi
+fi
```

Comment:
Should we not have a return 1 case here if vars are not set and/or message?

Reply:
Because we don't fall back

Reply:
that would prevent install, which is not ideal. it's better just to warn.